src/styles/main.scss

```diff
@@ -1,3 +1,65 @@
-body {
-  margin: 0;
+@import "./utils/variables.scss";
+@import "./utils/reset.scss";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  gap: $gap;
+  margin: 20px auto;
+  padding: $table-paddings;
+
+  $days:
+    mon 1,
+    tue 2,
+    wed 3,
+    thu 4,
+    fri 5,
+    sat 6,
+    sun 7;
+
+  @each $day, $day-number in $days {
+    &--start-day-#{$day} {
+      :first-child {
+        margin-left: (($day-number - 1) * $cell-size)
+          + ($gap * ($day-number - 1));
+      }
+    }
+  }
+
+  @for $month-end from 28 through 31 {
+    &--month-length-#{$month-end} {
+      :nth-child(n + #{$month-end}) {
+        display: none;
+      }
+    }
+  }
+
+  &__day {
+    height: $cell-size;
+    width: $cell-size;
+    border: 1px $numbers-color solid ;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $cell-color;
+    transition-duration: 0.5s;
```

Comment:
Let's specify the property for which we set transition duration. If you don't specify the property - it will affect in a bad way on the website performance